cmds/cpu/cpu9p_unix.go

```diff
@@ -0,0 +1,79 @@
+// Copyright 2022 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+//go:build !windows && !plan9
+// +build !windows,!plan9
+
+package main
+
+import (
+	"os"
+	"time"
+
+	"github.com/hashicorp/go-multierror"
+	"github.com/hugelgupf/p9/p9"
+	"golang.org/x/sys/unix"
+)
+
+// SetAttr implements p9.File.SetAttr.
+func (l *cpu9p) SetAttr(valid p9.SetAttrMask, attr p9.SetAttr) error {
+	var err error
+	// Any or ALL can be set.
+	// We started out with this:
+	// err = multierror.Append
+	// for each case. It seemed clean. But it returns a non-nil
+	// error, even if all arguments are nil. Interesting.
```

Comment:
Just a note: I'd expect the API for multierror to be that I'd have to check for its length > 0 instead of nil.

Reply:
multierror is not an array, so it would not have length. But it does have result.ErrorOrNil() so i'll use that.

Reply:
multierror is not as good an API as I hoped. Even their basic examples fail. I'll stick with it for not but it's a disappointment.